src/presence_analyzer/tests.py

```diff
@@ -86,6 +132,95 @@ def test_get_data(self):
             datetime.time(9, 39, 5)
         )
 
+    def test_group_by_weekday(self):
+        """
+        Test group by weekend.
+        """
+        excepted_result = [[], [0], [17088], [10800], [], [], []]
+        days = {
```

Comment:
You don't have to declare this. Put it in utils.group_by_weekday because you use it only once.